frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -96,9 +102,10 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
 
     const [volumeMode, setVolumeMode] = React.useState('');
     const onlyPvcSCs = (scObj: StorageClassResourceKind) =>
-      !snapshotClassResourceLoadError
-        ? scObj.provisioner.includes(snapshotClassResource?.driver)
-        : true;
+      !snapshotClassResourceLoadError &&
+      !scResourceLoadError &&
+      scObj.provisioner.includes(snapshotClassResource?.driver) &&
```

Comment:
To remove snapshotClassResource read,  I guess this line needs to be changed to:
```suggestion
      scObj.provisioner.includes(scResource?.provisioner) &&
```